apps/auth/src/users/users.service.ts

```diff
@@ -43,4 +52,20 @@ export class UsersService {
 
         return user;
     }
+
+    async getUser(getUserDto: GetUserDto): Promise<User> {
+        return this.usersRepository.findOne(getUserDto);
+    }
+
+    async transformUserToResponseUserDto(user: User) {
```

Comment:
Let's create a new type of file - mapper where we will keep such logic
users.mapper.ts